vue-frontend/src/components/jobs/CareerView.vue

```diff
@@ -133,7 +133,7 @@ export default {
           for (let i = 0; i < this.careers.length; i++) {
             var unique_id = this.careers[i].unique_id
             this.careers[i].detail_link = "/jobs/" + unique_id;
-            this.careers[i].job_link = "/jobs/apply/" + unique_id;
+            this.careers[i].job_link = "/jobs/apply-for-" + unique_id;
```

Comment:
I think this is fine, no need to make change. Keeping full component for id in path makes more sense as `jobs/` is reserved for detail page.